app.config.js

```diff
@@ -3,19 +3,19 @@ import { secrets } from './src/config/secrets';
 export default ({ config }) => {
   const namespace = config.slug;
   const sentryApi = secrets[namespace].sentryApi;
-  const hooks = sentryApi?.config
-    ? {
-        postPublish: [
-          {
-            file: 'sentry-expo/upload-sourcemaps',
-            config: sentryApi.config
-          }
+
+  const plugins = sentryApi.config
+    ? [
+        ...config.plugins,
+        [
+          '@sentry/react-native/expo',
+          { project: sentryApi.project, organization: sentryApi.organization }
         ]
-      }
-    : {};
+      ]
+    : [...config.plugins];
 
   return {
     ...config,
-    hooks
+    plugins
```

Comment:
is it correct to have the `plugins` next to `config` and not inside `config` as `config.plugins`?

Reply:
I changed the structure of `app.config` from `js` to `ts` and if there is a `sentryApi?.config` value, I added the sentry plugin with the `config.plugins.push` method.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1142/commits/676e132fe0fa49d875adacffbae3cea4f903ac70

Reply:
ouh yeah, very nice refactoring!